torch/jit/__init__.py

```diff
@@ -86,6 +86,34 @@ def module_lookup(names):
     return m
 
 
+def load_from_buffer(buffer):
```

Comment:
Hmm I'm not sure if we want to publicize this API too much, at least for now 😕

Reply:
A better API would be to make `torch.jit.load` behave more like `torch.load`. So it would accept both filenames and file-like objects, including `BytesIO`.

Reply:
I agree -- we want this api to eventually converge with `torch.load` so anything different is going to be more future BC work.